.github/workflows/auto-docs.yml

```diff
@@ -0,0 +1,37 @@
+name: Auto Generate Docs
+
+on:
+  schedule:
+    # Run every Monday at 9:00 AM UTC
```

Comment:
Rather than on a cron can we just have this run in commits to main? Then if we merge a PR it should run automatically.